requirements-docs.txt

```diff
@@ -0,0 +1,2 @@
+sphinx
+sphinx-panels
```

Comment:
do we not want to pin the versions?

Reply:
Not really, we are not using any particular feature that requires any version of sphinx. Regarding sphinx-panels we are using their most basic directive, I doubt they'll change it, and if they change it we probably want to notice it (which will happen on any PR).

Reply:
I understand your point but my view is unexpected changes can creep in that someone may not spot